### PR TITLE
Automatically collect call graph from `tdd_repro`

### PR DESCRIPTION
**Pull request by Domiii**
* This code primarly collects call graph data automatically from the `tdd_repro` command, with some customizations.
* I want to merge this, as I continue my manual investigation of `django__django-11299` ([PRO-864](https://linear.app/replay/issue/PRO-864/try-to-fix-failed-instance-django-django-11299)).
* https://linear.app/replay/issue/PRO-864/6a-test-initial-runtime-data-hypotheses#comment-8b2d0338

### CODE REVIEW

File: handpicked_logs/11299_1_first_success.log [hunk 1]
```diff
@@ -0,0 +1,2272 @@
+2024-10-01 11:28:38,309 WARNING ⚠ TDD is enabled. Agents have access to the test patch and will not try to reproduce the problem. ⚠
```
Comment (bhackett1024) [inline handpicked_logs/11299_1_first_success.log]: Can we remove these handpicked logs?  These are really large and don't seem necessary without associated docs.

**Review by bhackett1024 — APPROVED** [review]

File: handpicked_logs/11299_1_first_success.log (hunk 1 above)
Comment (Domiii) [reply handpicked_logs/11299_1_first_success.log]: I feel like the repo is the perfect place to keep track of large log files that sample our investigations.
Where do you suggest keeping them?

File: sweagent/environment/swe_env.py [hunk 2]
```diff
@@ -416,15 +416,36 @@ def init_container_prebake(self):
...
+            copy_anything_to_container(
+                self.container_obj,
+                host_asset_folder,
+                container_test_folder
+            )
+
+            # Inject test injecter:
+            # TODO: generalize the file location
+            #   (Choosing urls.py is a hack: we know that `urls.py` is one of the dynamic imports from runtests.py, so appending to it works.)
+            test_file_to_override = os.path.join(container_test_folder, "urls.py")
```
Comment (Domiii) [inline sweagent/environment/swe_env.py]: TODO: I still need to fix this before merging.

File: handpicked_logs/11299_1_first_success.log (hunk 1 above)
Comment (bhackett1024) [reply handpicked_logs/11299_1_first_success.log]: Hmm, to rephrase, what are these log files used for?  If we're just storing these log files here we could use google drive instead.

File: sweagent/environment/swe_env.py (hunk 2 above)
Comment (Domiii) [reply sweagent/environment/swe_env.py]: Disabled for now

**Pull request merged**